dashboard/config/scripts/course1.script

```diff
@@ -1,3 +1,6 @@
+is_course true
```

Comment:
what is the reasoning behind marking these as `is_course`? is this needed in order for the course to show up in the assignment selector dropdown?

Reply:
Maybe I misunderstood what these are. I thought these were similar to CourseA-F in that they're their own standalone things?

Right now I'm trying to just correctly create CourseVersions (and Courses, once we have that model) for things that should have a Course. It won't functionally affect anything until we change other parts of the code to use it, of course.

Eventually, things should have a Course to be eligible to show up in the dropdown, is what I imagine.

Reply:
OK, I agree with the goal that script which should show up in the dropdown should be marked with `is_course`. Mainly I wanted to agree on that, and it's up to you when to get into that state. makes sense that it doesn't need to be now since it doesn't affect anything user-visible yet.